modules/gltf/test/lib/extensions/EXT_feature_metadata.spec.js

```diff
@@ -394,3 +398,229 @@ test('gltf#EXT_feature_metadata - Should handle feature texture attributes', asy
   );
   t.end();
 });
+
+test('gltf#EXT_feature_metadata - Should handle arrays', async (t) => {
+  const binaryBufferData = [
```

Comment:
Could you write it in one line with `// prettier-ignore` comment? The buffer takes many lines although not readable.